pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "serenityff"
+version = "2.0.1"
+authors = [{ name = "rinikerlab", email = "mlehner@ethz.ch" }]
```

Comment:
Can you change my email to mlehner@proton.me? And the first email should probably be a permanent ETH address. Otherwise, this needs to be changed every 4 years